app/scripts/filters/findProject.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('findProject', function($filter) {
+    return function(items, searchString) {
+      // return early if no search string
+      if(!searchString) {
+        return items;
+      }
+
+      return _.filter(items, function(item) {
+        var searchStringLower = searchString.toLowerCase();
+
```

Comment:
I think it's just a lodash wrapper for `String.toLowerCase()`. We can stick with the native function.